games/h.yaml

```diff
@@ -13,17 +15,19 @@
 
 - name: Haunted House
   meta:
-    genre: [horror]
```

Comment:
I have made a statement about horror/survival horror as a standalone genre in this review already. This game is considered by name survival horror fans to be the first of its kind, even if is nothing but a very basic and early example of such. I'd like to keep it as `horror`.